packages/appeal-reply-service-api/src/models/blankModel.js

```diff
@@ -0,0 +1,103 @@
+const Mongoose = require('mongoose');
+const replySchema = require('./replySchema');
+
+const Model = Mongoose.model('Model', replySchema);
+const blankModel = new Model({
```

Comment:
I presume you're going to be changing this later?

Reply:
Depending on how we test.